src/app/[locale]/(other)/admin/notifications/page.tsx

```diff
@@ -0,0 +1,431 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+import { Button } from '@/components/ui/button';
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from '@/components/ui/select';
+import { Card, CardHeader, CardTitle, CardDescription, CardContent } from '@/components/ui/card';
+import { Badge } from '@/components/ui/badge';
+import { Bell, Send, Filter, Loader2, ExternalLink, RefreshCw, ChevronDown, ChevronUp, Mail, MessageSquare, Clock, CheckCircle2, XCircle, MapPin, Trash2 } from 'lucide-react';
+import { format } from 'date-fns';
+import { el } from 'date-fns/locale';
+import Link from 'next/link';
+import { Collapsible, CollapsibleContent, CollapsibleTrigger } from '@/components/ui/collapsible';
+import { NotificationMapDialog } from '@/components/admin/NotificationMapDialog';
+
+export default function AdminNotificationsPage() {
+    const [notifications, setNotifications] = useState<any[]>([]);
```

Comment:
We wouldn't need to use `any` here if we properly defined the related type in `src/lib/db/notifications.ts` (as mentioned in my file comment in `src/lib/db/notifications.ts`). This data is returned from `/api/admin/notifications` which uses `getNotificationsForAdmin()`